dts/bindings/display/backlight.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2024, Martin Kiepfer
+# SPDX-License-Identifier: Apache-2.0
+
+include: [base.yaml]
+
+properties:
+  init-level:
+    type: int
+    default: 80
+    description: Initial value of the Backlight.
```

Comment:
not clear what the value means from the DT description (I can see it becomes the pulse in code, but user should not have to look for that)